Cargo.toml

```diff
@@ -26,6 +26,7 @@ members = [
   "bindings/nodejs",
   "bindings/object_store",
   "bindings/python",
+  "bindings/java/src/main/rust/opendal-java",
```

Comment:
Is it possible that we keep the rust project at `bindings/java`?

Reply:
Yes, this is the way of Java doing, it requires a or to rust-maven-plugin to do so, I will refactor it later.